README.md

```diff
@@ -203,3 +182,26 @@ Below are some of those features.
 - Add optional one-time setup phase for applications
 - Add option to download bounty bundle
 - Sandbox applications with Hypervisor
+
+## Debugging
+
+When running BugLess locally, you might want to perform some operations that would otherwise be impossible in a production environment.
+To this end, we advise you to install the [Foundry](https://book.getfoundry.sh/getting-started/installation) toolkit.
+
+### Time travel
+
+When testing sponsor withdrawals, it's handy to be able to instantly advance time past the expiry date of some bounty.
+The following command advances time in 30 days, expressed in seconds.
+
+```sh
+cast rpc evm_increaseTime $((60*60*24*30))
+```
+
+### Funding a wallet
+
+In order to publish transactions, you need some Ether.
+The following command sets the balance of address `0xf39Fd...92266` to 1 Ether, expressed in Wei.
+
+```sh
+cast rpc anvil_setBalance 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 $(cast to-wei 1 ether)
```

Comment:
hey @guidanoli , `anvil` automatically funds 10 accounts (including the one above) with 10000 ETH to begin with, right?

Reply:
Yes, but if you don't want to use the `test [test ...] test junk` mnemonic, but your own test Metamask wallet, you can do so by funding it like so.